crate/server/src/database/cached_sqlite_struct.rs

```diff
@@ -19,7 +19,7 @@ pub struct KMSSqliteCacheItem {
     /// The handler to the sqlite
     sqlite: Arc<Pool<Sqlite>>,
     /// They key of the sqlite
-    key: Key<32>,
+    key: [u8; 32],
```

Comment:
Can you use a constant from somewhere instead of `32`?

Reply:
done in https://github.com/Cosmian/kms/pull/16/commits/2926f421f66d1b589c1e33262ab0dd1d85a375aa